packages/server-render/server-sink.js

```diff
@@ -30,6 +33,16 @@ export class ServerSink {
     this.htmlById[id] = "";
     this.appendToElementById(id, html);
   }
+
+  renderDocument(html){
+      // Extract head
+      const head = HEAD_REGEX.exec(html)[1];
+      this.appendToHead(head);
+
+      // Extract body
+      const body = BODY_REGEX.exec(html)[1];
+      this.appendToBody(body);
```

Comment:
Honestly, there's nothing happening here that a user of the `ServerSink` couldn't already do by calling `sink.appendToHead` and `sink.appendToBody`, and that code would be much easier to understand than `sink.renderDocument`.